dohq_artifactory/admin.py

```diff
@@ -93,9 +94,14 @@ def _create_and_update(self):
             headers={"Content-Type": "application/json"},
             auth=self._auth,
         )
-        r.raise_for_status()
-        rest_delay()
-        self.read()
+        try:
+            r.raise_for_status()
+        except requests.exceptions.HTTPError as e:
+            print("Response text: {}".format(e.resonse.text))
```

Comment:
Probable, the best way to say something to people who use you library it's to use [logging](https://docs.python.org/3/library/logging.html), not `print`. It's too hard to configure `print` in a application, not just cli-script.